### PR TITLE
bulid web site html css 

### PR DESCRIPTION
**Pull request by nesmamanasra**
- In the index. Html file, the tags required for the design in terms of the sections and other required things have been placed.
- In the style.css file, the required formats were developed using the class names and the css properties were put in place to format the page and show it appropriately
prommm

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,8 +8,40 @@
...
+        <header>
+            <h1>
+                <span>rock</span>
+                <span>paper</span>
+                <span>scissors</span>
+            </h1>
+            <!-- <div class="score-contanier">
+                <h4>score</h4>
+                <p>0</p>
+            </div> -->
```
Comment (bayan-404) [inline index.html]: delete the commented code

File: style.css [hunk 2]
```diff
@@ -1,3 +1,88 @@
...
+ }
+ header{
+     border:3px solid #000;
+     display: flex;
+     border-radius: 20px;
+     justify-content: space-between;
+     padding: 25px;
+     margin-bottom: 50px;
+ }
+ h1{
```
Comment (bayan-404) [inline style.css:22]: for the next time >> you should always use classes don't use the tags for the style

**Review by bayan-404 — APPROVED** [review]

File: style.css [hunk 3]
```diff
@@ -1,3 +1,88 @@
 *{
     box-sizing: border-box;
 }
+body{
+    background-color: #ccc;
+  background-color: #ccc;
```
Comment (saharAdem) [inline style.css:5]: remove this repeated line.

**Review by saharAdem — CHANGES_REQUESTED** [review]

File: index.html [hunk 4]
```diff
@@ -8,8 +8,40 @@
...
+             You
+             <span class="result-score">0</span>
+ 
+         </div>
+         <div>
+             Computer
+             <span class="result-score">0</span>
+         </div>
+     </div>
+     <!-- <div class="result-selection winner">✋</div>
```
Comment (Rahaf-96) [inline index.html]: delete these comments, please

File: style.css (hunk 2 above)
Comment (nesmamanasra) [reply style.css:22]: ok

File: index.html (hunk 1 above)
Comment (nesmamanasra) [reply index.html]: I did what was required

File: style.css (hunk 3 above)
Comment (nesmamanasra) [reply style.css:5]: I did what was required of me

File: index.html (hunk 4 above)
Comment (nesmamanasra) [reply index.html]: I did what was required of me

**Review by bayan-404 — APPROVED** [review]

**Review by saharAdem — APPROVED** [review]

**Review by Rahaf-96 — APPROVED** [review]
You did an amazing job😍 keep it up✌🏻

**Pull request merged**